src/components/PhotosGallery/PhotosGallery.jsx

```diff
@@ -0,0 +1,124 @@
+import Image from "next/image"
+import React, { useEffect, useState } from "react"
+
+function PhotosGallery(props) {
+  const [file, setFile] = useState(null)
+
+  const onChangeImageHandler = (e) => {
+    if (e.target.files[0]) {
+      const reader = new FileReader()
+      reader.onload = () => {
+        setFile(reader.result)
+      }
+      reader.readAsDataURL(e.target.files[0])
+      e.target.value = ""
+    }
+  }
+
+  useEffect(() => {}, [file])
+
+  function dropLargeImage() {
+    return (
+      <div className='w-full'>
+        <div className='border border-solid bg-gray-300 border-black rounded relative '>
+          <input
+            type='file'
+            className='cursor-pointer relative opacity-0 w-full p-14 lg:p-24 md:p-24'
+            onChange={onChangeImageHandler}
+          ></input>
+          <div className='w-12 absolute top-10 md:top-20 lg:top-20 right-0 left-0 m-auto'>
+            <Image
+              src='gallery.svg'
+              width={100}
+              height={100}
+              alt='drop large image'
+            />
+          </div>
+        </div>
+      </div>
+    )
+  }
+
+  function dropImage() {
+    return (
+      <div className='w-full'>
+        <div className='border border-solid bg-gray-300 border-black rounded relative '>
+          <input
+            type='file'
+            className='cursor-pointer relative opacity-0 w-full p-8 lg:p-10 md:p-10'
+            onChange={onChangeImageHandler}
+          ></input>
+          <div className='w-8 absolute top-8 right-0 left-0 m-auto'>
+            <Image
+              src='gallery.svg'
+              width={100}
+              height={100}
+              alt='drop image'
+            />
+          </div>
+        </div>
+      </div>
+    )
+  }
+
+  return (
+    <div className='flex flex-col lg:flex-col md:flex-row w-full justify-center'>
+      <div className='lg:m-0 my-auto w-full justify-center'>
+        {props.largimage ? (
+          <Image
+            className='h-full w-full lg:w-full lg:h-full md:w-full object-cover'
+            src={props.largimage}
```

Comment:
It would be best to use the optional chaining operator ("?." or ".") when accessing the "props" object, so that you can avoid errors when the "props" object is null or undefined.
`src={props?.largimage}`